### ISSUE TITLE
Improve the color contrast of pagination links 

### ISSUE THREAD

**Issue by cindyli** [inclusive-design/wecount.inclusivedesign.ca#397]
## Describe the bug

Running the lighthouse report discovers the color contrast of pagination links is not sufficient on mobile devices. Comparing the implementation on dev site and the figma design, the background color should be 465D8B while the website is using 5879a0. The background color should be adjusted to match the design.

## To Reproduce

1. Check [the report of "lighthouse CI - Mobile"](https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/293/checks?check_run_id=1130461284) generated by [the lighthouse CI pull request](https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/293);
2. Accessibility score of a few pagination links is dropped from 100 to 98.
